docs/source/configuration/logging.rst

```diff
@@ -15,3 +15,22 @@ the logging, simply enable ``TRACE`` logging for
 
 If you are using ``CassandraKeyValueService``, you can use additional tracing for deeper analysis.
 For more information, see :ref:`enabling-cassandra-tracing`.
+
+Debug Logging for Multiple Timestamp Services Error
+===================================================
+
+From version 0.22, it is recommended that you send logging related to the timestamp service to a separate appender.
+To do this, add the following to your logging configuration:
+
+.. code:: yaml
+
+    logging:
+      loggers:
+        com.palantir.timestamp:
```

Comment:
We probably just want the `com.palantir.timestamp.DebugLogger` stuff to go to a different file, right?